### ISSUE TITLE
Could not get IMDB info - due to 308 redirect

### ISSUE THREAD

**Issue by GregSawin** [SickChill/sickchill#8333]
Version: [2023.1.2]
OS: Windows 10
Browser: Chrome
What you did: Looked at the logs after the recent update
What happened: I saw many errors mentioning a 308 Redirect because the urls don't have a "/" between "find" and "?".  
What you expected: Not to see those errors
Logs:
```
2023-01-03 13:48:08 DEBUG :: MAIN :: Loading initial show list
2023-01-03 13:48:08 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/find?q=Wilmore+%282020%29&s=tt', 'proxy': '', 'exception type': 'IOError', 'original exception': <HTTPError 308: 'Permanent Redirect'>}
2023-01-03 13:48:08 INFO :: MAIN :: 387995: Unable to find IMDb show info in the database
2023-01-03 13:48:08 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/find?q=World%27s+Funniest+Animals&s=tt', 'proxy': '', 'exception type': 'IOError', 'original exception': <HTTPError 308: 'Permanent Redirect'>}
```

I believe the solution is either to follow the 308 redirects or to update all calls in the format of "https://www.imdb.com/find?xyz" to "https://www.imdb.com/find/?xyz" with a new slash before the querystring.

**Comment by davidreverett** [comment]
Seeing similar post upgrade: 

2023-01-04 18:07:03 INFO :: CHECKVERSION :: Checking for updates using PIP
2023-01-04 18:07:03 INFO :: WEBSERVER :: Starting SickChill on http://0.0.0.0:8081/
2023-01-04 18:07:03 INFO :: MAIN :: Forcing web server to port 8081
2023-01-04 18:07:03 INFO :: MAIN :: Starting SickChill [2023.1.2] using 'C:\SickChill\Data\config.ini'
2023-01-04 18:07:03 INFO :: MAIN :: 428423: Unable to find IMDb show info in the database
2023-01-04 18:07:03 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/find?q=MILF+Manor&s=tt', 'proxy': '', 'exception type': 'IOError', 'original exception': }
2023-01-04 18:07:03 INFO :: MAIN :: 345678: Unable to find IMDb show info in the database
2023-01-04 18:07:03 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/find?q=Ch%C3%A2teau+DIY&s=tt', 'proxy': '', 'exception type': 'IOError', 'original exception': }
2023-01-04 18:07:03 INFO :: MAIN :: 428633: Unable to find IMDb show info in the database
2023-01-04 18:07:03 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/find?q=%22Escape+to+the+Ch%C3%A2teau%3A+Secret+France%22+%280%29&s=tt', 'proxy': '', 'exception type': 'IOError', 'original exception': }
2023-01-04 18:07:01 INFO :: EVENT-QUEUE :: Restarting SickChill with ['C:\\SickChill\\Python3\\tools\\python.exe', 'C:\\SickChill\\Python3\\tools\\lib\\site-packages\\SickChill.py', '--nolaunch', '--port=8081', '--datadir=C:\\SickChill\\Data']

**Comment by miigotu** [comment]
These are not errors, the log level is INFO. IOError is just the name of a python class. It just can't find a singular result from the search, either because the show isn't on IMDb or it's called something different or there are many other shows with a similar name.

**Comment by kailia** [comment]
OK it may be only "Info" level messages, but it's still weird that it can't find singular result, while all the links below indeed give singular results when you open the URLs:

2023-03-08 08:43:00 INFO :: MAIN :: 393148: Unable to find IMDb show info in the database
2023-03-08 08:43:00 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/title/tt13821126/plotsummary', 'proxy': '', 'exception type': 'IOError', 'original exception': }
2023-03-08 08:42:58 INFO :: MAIN :: 387698: Unable to find IMDb show info in the database
2023-03-08 08:42:58 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/title/tt9648870/plotsummary', 'proxy': '', 'exception type': 'IOError', 'original exception': }
2023-03-08 08:42:56 INFO :: MAIN :: 311954: Unable to find IMDb show info in the database
2023-03-08 08:42:56 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/title/tt5420376/plotsummary', 'proxy': '', 'exception type': 'IOError', 'original exception': }
2023-03-08 08:42:54 INFO :: MAIN :: 365026: Unable to find IMDb show info in the database
2023-03-08 08:42:54 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/title/tt0489974/plotsummary', 'proxy': '', 'exception type': 'IOError', 'original exception': }
2023-03-08 08:42:52 INFO :: MAIN :: 418930: Unable to find IMDb show info in the database
2023-03-08 08:42:52 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/title/tt19724076/plotsummary', 'proxy': '', 'exception type': 'IOError', 'original exception': }
2023-03-08 08:42:51 INFO :: MAIN :: 421636: Unable to find IMDb show info in the database
2023-03-08 08:42:51 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/title/tt21057450/plotsummary', 'proxy': '', 'exception type': 'IOError', 'original exception': }
2023-03-08 08:42:49 INFO :: MAIN :: 373451: Unable to find IMDb show info in the database
2023-03-08 08:42:49 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/title/tt10369484/plotsummary', 'proxy': '', 'exception type': 'IOError', 'original exception': }
2023-03-08 08:42:48 INFO :: MAIN :: 403752: Unable to find IMDb show info in the database
2023-03-08 08:42:48 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/title/tt14761860/plotsummary', 'proxy': '', 'exception type': 'IOError', 'original exception': }
2023-03-08 08:42:46 INFO :: MAIN :: 400779: Unable to find IMDb show info in the database
2023-03-08 08:42:46 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/title/tt0808491/plotsummary', 'proxy': '', 'exception type': 'IOError', 'original exception': }
2023-03-08 08:42:45 INFO :: MAIN :: 403623: Unable to find IMDb show info in the database
2023-03-08 08:42:45 INFO :: MAIN :: Could not get IMDB info: {'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/title/tt18949726/plotsummary', 'proxy': '', 'exception type': 'IOError', 'original exception': }
2023-03-08 08:42:43 INFO :: MAIN :: 116381: Unable to find IMDb show info in the database

**Comment by BKSteve** [comment]
This `plotsummary` error occurs on my current `master` installation on my NAS but does not occur on my Linux PyCharm installation which is on  `develop`

**Comment by miigotu** [comment]
Yes there was an updated dependency that now handles 308 and a few other new http return codes. It should be time for a new release soon.

**Comment by BKSteve** [comment]
> It should be time for a new release soon.

Should probably do an update to SynoComm package too at the next release.
Give me a heads up when and I can pre test and be ready.

**Comment by miigotu** [comment]
Fixed in cinemagoer 6.5, will be included in the next release.
